### PR TITLE
Update drand endpoints

### PR DESCRIPTION
**Pull request by hsanjuan**
* Remove incentinet (should not be used already)
* Add cloudflare endpoint.
* Switch PL endpoints to HTTP instead of HTTPS.

### CODE REVIEW

File: build/drand.go [hunk 1]
```diff
@@ -32,9 +32,10 @@ const (
 var DrandConfigs = map[DrandEnum]dtypes.DrandConfig{
 	DrandMainnet: {
 		Servers: []string{
-			"https://api.drand.sh",
-			"https://api2.drand.sh",
-			"https://api3.drand.sh",
+			"http://api.drand.sh",
```
Comment (hsanjuan) [inline build/drand.go:35]: I do not see any potential negative impact of doing this, unless GFW would prefer to block HTTP or delay traffic to cloudfront (?). If there's a concern, I can leave things as they are.

**Comment by nikkolasg** [comment]
My opinion: I would tend to keep HTTPs for three reasons:
1. biggest one: User is sure that the value comes from the drand node and hasn't been MitM, externally tampered with
2. Internet in general is tending towards full https everywhere - it's like using IPv6, if we can we should
3. If the goal is to reduce bandwitdth, we should put most of our efforts into making sure gossipsub is used as much as possible by Lotus node, since that's our primary use case.

File: build/drand.go (hunk 1 above)
Comment (magik6k) [reply build/drand.go:35]: I don't see any advantage to using http. If we can use https, we should use it

**Comment by hsanjuan** [comment]
Sorry, to highlight advantages: HTTPS has more latency. There is also a cost associated with running the endpoints (HTTP requests costs less).

MiTM is not a risk here, as randomness is verified after fetching.

**Review by willscott — APPROVED** [review]
i don't think i have a strong preference on the http/https thing.
adding cloudflare is good.

**Review by magik6k — COMMENTED** [review]
Tests are failing

**Comment by hsanjuan** [comment]
Thanks @willscott

**Comment by magik6k** [comment]
Can't sync a fresh node with this PR applied, getting errors like
```
2020-10-04T17:40:47.513+0200    INFO    chain   chain/sync.go:706       block validation        {"took": 0.303595103, "height": "1", "age": 3519617.513044093}
2020-10-04T17:40:47.513+0200    ERROR   chain   chain/sync.go:1470      failed to validate tipset: validating block bafy2bzacechdx6xd62lcyy7rnyc4uxcxhuwqslcxfvj77fxlwafij3nhzchpy:
    github.com/filecoin-project/lotus/chain.(*Syncer).ValidateTipSet.func1
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:620
  - 1 error occurred:
        * failed to validate blocks random beacon values:
    github.com/filecoin-project/lotus/chain.(*Syncer).ValidateBlock.func9
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:888
  - expected final beacon entry in block to be at round 118327, got 81086:
    github.com/filecoin-project/lotus/chain/beacon.ValidateBlockValues
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/beacon/beacon.go:82


2020-10-04T17:40:47.513+0200    ERROR   chain   chain/sync_manager.go:421       sync error: collectChain failed:
    github.com/filecoin-project/lotus/chain.(*Syncer).Sync
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:570
  - collectChain syncMessages:
    github.com/filecoin-project/lotus/chain.(*Syncer).collectChain
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:1695
  - message processing failed:
    github.com/filecoin-project/lotus/chain.(*Syncer).syncMessagesAndCheckState.func1
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:1471
  - validating block bafy2bzacechdx6xd62lcyy7rnyc4uxcxhuwqslcxfvj77fxlwafij3nhzchpy:
    github.com/filecoin-project/lotus/chain.(*Syncer).ValidateTipSet.func1
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:620
  - 1 error occurred:
        * failed to validate blocks random beacon values:
    github.com/filecoin-project/lotus/chain.(*Syncer).ValidateBlock.func9
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/sync.go:888
  - expected final beacon entry in block to be at round 118327, got 81086:
    github.com/filecoin-project/lotus/chain/beacon.ValidateBlockValues
        /home/magik6k/github.com/filecoin-project/go-lotus/chain/beacon/beacon.go:82

```

**Comment by willscott** [comment]
It looks like the chain up to block 51000 [was using the incentinet](https://github.com/filecoin-project/lotus/pull/4125/files#diff-63225929e8456070a0375391e7dd0664L15) for randomness.
While it doesn't need to contact the incentinet endpoint (I previously synced happily with no valid endpoints.) It will need the [root of trust](https://github.com/filecoin-project/lotus/pull/4125/files#diff-f4939db398f92f6efd739457eeddc5f3L81) to validate that the beacons encoded in those previous chain blocks are valid.

**Review by raulk — COMMENTED** [review]
Is the test-vectors submodule update intentional?

**Comment by hsanjuan** [comment]
> Is the test-vectors submodule update intentional?

pretty sure not, fixed.

**Comment by hsanjuan** [comment]
> Can't sync a fresh node with this PR applied, getting errors like

I'm syncing ok from scratch, at least 1300 rounds already, so I guess @willscott solved that issue.

**Comment by willscott** [comment]
@magik6k this is ready for another merge attempt

**Review by Kubuxu — APPROVED** [review]

**Review by magik6k — APPROVED** [review]

**Pull request merged**